pyproject.toml

```diff
@@ -270,12 +269,19 @@ exclude=[
 
 [tool.ruff.per-file-ignores]
 "__init__.py" = ["E402", "F401", "F403"]
+"astropy/coordinates/angle_formats.py" = ["D415"]
+"astropy/coordinates/angle_lextab.py" = ["D415"]
+"astropy/coordinates/angle_parsetab.py" = ["D415"]
```

Comment:
The `*_lextab.py` and `*_parsetab.py` files are automatically generated, so I would exclude them from all code style checks, not just D415.

Reply:
✅  I've added them to the exclusion list, removing them from the per-file ignores. Ruff will not run on lext/parsetab at all.